tests/test_scalars.py

```diff
@@ -314,73 +318,55 @@ def test_interval_cmp():
 
     interval_index_int = pd.IntervalIndex([interval_i])
     check(
-        assert_type(interval_index_int >= interval_i, np_ndarray_bool),
-        np.ndarray,
-        np.bool_,
+        assert_type(interval_index_int >= interval_i, np_1darray[np.bool]),
+        np_1darray[np.bool],
     )
```

Comment:
for these tests, put back the `np.bool_` argument.  (This one and all the ones through line 384 of the previous version